src/service.toit

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023 Toitware ApS. All rights reserved.
```

Comment:
In a number of cases, we've put a file like this in `src/api/`. This package may be different, because it is almost entirely the API. But there is a difference between the provider and the API that may warrant some kind of split.

Reply:
done.